system-addon/common/Reducers.jsm

```diff
@@ -6,8 +6,8 @@
 const {actionTypes: at} = ChromeUtils.import("resource://activity-stream/common/Actions.jsm", {});
 const {Dedupe} = ChromeUtils.import("resource://activity-stream/common/Dedupe.jsm", {});
 
-const TOP_SITES_DEFAULT_ROWS = 2;
-const TOP_SITES_MAX_SITES_PER_ROW = 6;
+const TOP_SITES_DEFAULT_ROWS = 1;
```

Comment:
After looking at the remaining use of `TOP_SITES_DEFAULT_ROWS`, it looks like it's either wrong (`HighlightsFeed`) or only used in tests, where we could just separately define. Let's clean this up separate from this PR

Reply:
Hmm. Looking at `HighlightsFeed` again I was wrong about Top Sites not getting deduped from Highlights. And yeah, the use there of the default rows is wrong. I think I'll ask one more time if we can remove that deduping 😄 . Seems like it should only come into play on newish profiles anyway.